src/libsync/clientsideencryption.cpp

```diff
@@ -1667,7 +1667,9 @@ bool EncryptionHelper::fileEncryption(const QByteArray &key, const QByteArray &i
             return false;
         }
 
+#ifdef QT_DEBUG
         qCDebug(lcCse) << "Encrypting " << data;
```

Comment:
I guess it could be a better idea to just remove it

Reply:
Maybe it is useful for debugging. Yeah or I just remove it.